src/main/flight/servos.c

```diff
@@ -296,20 +296,54 @@ STATIC_UNIT_TESTED void forwardAuxChannelsToServos(uint8_t firstServoIndex)
     }
 }
 
+static void biquadFilterInitd1d2(biquadFilter_t *filter, float sample)
```

Comment:
The function calculates sensible initial values for the biquad variables d1 and d2 based on the first sample. Zeros (default) are not a sensible values as the first few samples then applied to the filter will create erratic output. Possibly overshoot and ringing, but that depends on the Q-value of the filter. As far as what d1 and d2 actually are and how biquad filters work - I am clueless. In my opinion this function should be moved to filter.c at some point. I really have no good name for the function.